packages/select/stories/SelectField.stories.tsx

```diff
@@ -14,7 +15,10 @@ export default {
   title: 'Form Components/Select',
   parameters: {
     docs: {
-      // page: README,
+      description: {
+        component:
+          'The same as Select but with a Label that appears above the input and a Feedback that appears below the input.',
```

Comment:
The pattern we've been using is for the top level description to be the description for the entire package. Unfortunately with the way these stories are currently setup it seems this will list the selectfield description right underneath the package title ("select") making it a bit confusing.

I think it'd be best to move the selectfield story to the select.stories file and then add the description to the story itself ( see https://github.com/Availity/availity-react/blob/master/packages/form/src/Form.stories.tsx#L154-L159 ). Then we could get rid of this extra stories file.

Also, it'd be more accurate to say "Select with a Label and Feedback" as opposed to "the same as select but..." since it uses the select component.